cmd/root.go

```diff
@@ -354,8 +357,18 @@ func initConfig() {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	go handleInterrupt()
 	app = application.New()
 	rootCmd := NewRootCmd()
 	err := rootCmd.Execute()
 	cobrautils.HandleErrors(err)
 }
+
+func handleInterrupt() {
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
+	sig := <-sigChan
+	fmt.Println()
+	fmt.Println("received signal:", sig.String())
```

Comment:
is it debug println? Pls feel free to ignore this comment

Reply:
it is not debug println. but no other output mechanism (eg logging) is generally available here